moncli/types.py

```diff
@@ -211,4 +214,78 @@ def validate_timezone(self, value):
         try:
             pytz.timezone(value)
         except (UnknownTimeZoneError):
-            raise ValidationError('Unknown time zone "{}".'.format(value))
\ No newline at end of file
+            raise ValidationError('Unknown time zone "{}".'.format(value))
+
+class StatusType(MondayType):
+    native_type = str
+    allow_casts = (int, str)
+    null_value = {}
+    def __init__(self, id: str = None, title: str = None,data_mapping:dict =None ,  *args, **kwargs):
+        if data_mapping:
+            values= [value for value in data_mapping.values()]        
+            self.native_type = values[0].__class__
+            self.choices = values
+        self._data_mapping =  data_mapping 
+        super().__init__(id=id, title=title, *args, **kwargs)
+    
+    def _process(self, value):
+        labels = self.metadata['labels']
+        if self.native_type == str:
```

Comment:
Best practices for clean code state that we should exit sooner than later.  Let's clean this up by reversing this statement to the following.

```python
if self.native_type != str:
    return value
```

The last point of the document specified that if the native_type was an Enum class instance, then we simply return the value.  This way, we can verify that if it is, we leave right away.  If it is not, we can assume that it is a str without making any additional checks.